chat-rh.py

```diff
@@ -0,0 +1,34 @@
+from llama_cpp import Llama
+
+class Chat:
+
+    n_ctx = 2048
+
+    def __init__(self) -> None:
+        self.chat_history = [
+                {"role": "system", "content": """You are a helpful assistant that is comfortable speaking
+                 with C level executives in a professional setting."""},
+                ]
+        self.llm = Llama(model_path="/locallm/nested/dir/llama-2-7b-chat.Q4_K_M.gguf", 
```

Comment:
that #L12 is the only line that is different from chat.py

Reply:
Might be easier to just add `model_path` parameter to the the chat class `__init__`, instead of managing another file.

Reply:
yup was thinking that but also was being lazy